build_tools/azure/install.sh

```diff
@@ -120,7 +120,21 @@ except ImportError:
 "
 python -m pip list
 
-# Use setup.py instead of `pip install -e .` to be able to pass the -j flag
-# to speed-up the building multicore CI machines.
-python setup.py build_ext --inplace -j 3
-python setup.py develop
+if [[ "$BUILD_WITH_ICC" == "true" ]]; then
+    wget https://apt.repos.intel.com/intel-gpg-keys/GPG-PUB-KEY-INTEL-SW-PRODUCTS-2023.PUB
+    sudo apt-key add GPG-PUB-KEY-INTEL-SW-PRODUCTS-2023.PUB
+    rm GPG-PUB-KEY-INTEL-SW-PRODUCTS-2023.PUB
+    sudo add-apt-repository "deb https://apt.repos.intel.com/oneapi all main"
+    sudo apt-get update
+    sudo apt-get install intel-oneapi-icc
+    source /opt/intel/inteloneapi/setvars.sh
+
+    python setup.py build_ext --compiler=intelem -i -j 3 build_clib --compiler=intelem
```

Comment:
Please add a comment to explain that the `build_clib` command is required to build libsvm that is registered as a library instead of an extension in our distutils config.